src/fields/components/InternalField.tsx

```diff
@@ -1,52 +1,59 @@
 import React, { ReactNode } from 'react'
 import styled from 'styled-components'
 
-import { Text } from '../Text'
-import { Box } from '../Box'
-import { CommonFieldTypes } from './types/commonFieldTypes'
+import { Text } from '../../Text'
+import { Box } from '../../Box'
+import { InternalCommonFieldProps } from '../commonFieldTypes'
 
-interface FieldProps extends CommonFieldTypes {
+interface InternalFieldProps extends InternalCommonFieldProps {
   children: ReactNode
+  className?: string
   assistiveText?: string
+  htmlFor?: string
+  fieldType?: 'field' | 'fieldset'
 }
 
-export const Field = ({
+export const InternalField = ({
```

Comment:
Created an internal component that can be used on both `Field` and `Fieldset` so now they share the same UI and logic

Reply:
Love this